### PR TITLE
Handle exit code when calling resolve-import-conflicts

### PR DESCRIPTION
**Pull request by mernst**

### CODE REVIEW

**Comment by benedikt-schesch** [comment]
@mernst You don't think it's better like this in one script?

**Comment by benedikt-schesch** [comment]
@mernst I am not a fan of the structured you proposed with calling another script, it creates too much dependencies between the scripts and too much calling around in my opinion. I looked at the script and made the return values explicit because it makes it easier to understand what we are trying to do here. For me in the current way it is a lot more intuitive.
I have a problem with `gitmerge_resolve.sh`, the scripts only seems to remove some conflict markers but the exit statement seems to preserve the old return value so in the end it will still be marked as a merge failure, thus this script is not really doing anything since it won't be tested due to merge conflict. I am not 100% sure what the desired behavior is here. 
That is why I think making the return values explicit is important.

File: src/scripts/merge_tools/gitmerge_ort.sh [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-# usage: ./gitmerge-ort.sh <clone_dir> <branch-1> <branch-2>
+# usage: ./gitmerge_ort.sh <clone_dir> <branch-1> <branch-2>
 
 MERGE_SCRIPTS_DIR="$(cd "$(dirname "$0")" && pwd -P)"
 clone_dir=$1
 branch1=$2
 branch2=$3
 strategy="-s ort"
-"$MERGE_SCRIPTS_DIR"/gitmerge.sh "$clone_dir" "$branch1" "$branch2" "$strategy"
+if ! "$MERGE_SCRIPTS_DIR"/gitmerge.sh "$clone_dir" "$branch1" "$branch2" "$strategy"; then
```
Comment (mernst) [inline src/scripts/merge_tools/gitmerge_ort.sh:10]: @benedikt-schesch Why is this necessary?  The script should exit with the same status as the last command in it, so this `if` construct seems unnecessary (throughout).

**Comment by mernst** [comment]
I was trying to leave AST-Merging-Evaluation functionality (like printing "Conflict" and running `merge commit --abort` out of `resolve-import-conflicts` which is independently useful and exists at https://github.com/plume-lib/git-scripts.  We might eventually want to use the version from there.  In the meanwhile, see https://github.com/benedikt-schesch/AST-Merging-Evaluation/pull/218, which is a better alternative to this pull request.

**Comment by benedikt-schesch** [comment]
#218 Is better
